ui/src/literature/containers/LiteratureSeminars.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { useCallback, useEffect, useMemo } from 'react';
+import { Col } from 'antd';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { List } from 'immutable';
+
+import { LITERATURE_SEMINARS_NS } from '../../search/constants';
+import { LOCAL_TIMEZONE } from '../../common/constants';
+import SeminarItem from '../../seminars/components/SeminarItem';
+import ResultsContainer from '../../common/containers/ResultsContainer';
+import PaginationContainer from '../../common/containers/PaginationContainer';
+import { searchBaseQueriesUpdate } from '../../actions/search';
+import EmptyOrChildren from '../../common/components/EmptyOrChildren';
+
+function LiteratureSeminars({ onBaseQueryChange, recordId, seminars }) {
+  const baseQuery = useMemo(
+    () => ({
+      q: `literature_records.record.$ref:${recordId}`,
+    }),
+    [recordId]
+  );
+
+  useEffect(
+    () => {
+      onBaseQueryChange(LITERATURE_SEMINARS_NS, baseQuery);
+    },
+    [baseQuery, onBaseQueryChange]
+  );
+
+  const renderSeminarItem = useCallback(
+    result => (
+      <SeminarItem
+        metadata={result.get('metadata')}
+        selectedTimezone={LOCAL_TIMEZONE}
+        enableActions={false}
+      />
+    ),
+    []
+  );
+
+  return (
+    <Col span={24}>
```

Comment:
Is this really needed? it's strange to root of the component to be `Col` without `Row`